src/main/java/org/folio/inventory/dataimport/handlers/matching/loaders/AbstractLoader.java

```diff
@@ -85,6 +88,21 @@ protected AbstractLoader(Vertx vertx) {
     return future;
   }
 
+  /**
+   * Creates paging parameters for entities loading.
+   * If matching result of current matching can be processed by next profile than returns parameters with limit = 90.
+   * Otherwise, for performance needs returns paging parameters with limit = 2, which is
+   * a minimum value that is necessary to get target record or identify whether multiple match result occurred.
+   *
+   * @param multiMatchResultParams - identifies whether to return paging parameters for multiple matching
+   * @return {@link PagingParameters}
+   */
+  private PagingParameters buildPagingParameters(boolean multiMatchResultParams) {
+    // currently, limit = 90 is used because of constraint for URL size that is used for processing multi-match result
+    // in scope of https://issues.folio.org/browse/MODDICORE-251 a new approach will be introduced for multi-matching result processing
+    return multiMatchResultParams ? new PagingParameters(90, 0) : new PagingParameters(2, 0);
```

Comment:
@RuslanLavrov I think it would be better to extract this value (90) to the specific constant via a logic-based name. What do you think?

Reply:
Hi @VRohach, moved to constant